dotcom-rendering/src/model/enhance-H3s.test.ts

```diff
@@ -0,0 +1,200 @@
+import { blockMetaData } from '../../fixtures/manual/block-meta-data';
+import { enhanceH3s } from './enhance-H3s';
+
+describe('Enhance H3s', () => {
+	it('does set a h3 if there is more than one strong tag', () => {
+		const input: Block[] = [
+			{
+				...blockMetaData,
+				elements: [
+					{
+						_type: 'model.dotcomrendering.pageElements.TextBlockElement',
+						elementId: 'mockId',
+						html: '<p><strong>Strong 1</strong> <strong>Strong 2</strong></p>',
+					},
+				],
+			},
+		];
+		const expectedOutput: Block[] = [
+			{
+				...blockMetaData,
+				elements: [
+					{
+						_type: 'model.dotcomrendering.pageElements.TextBlockElement',
+						elementId: 'mockId',
+						html: '<h3>Strong 1 Strong 2</h3>',
+					},
+				],
+			},
+		];
+		expect(enhanceH3s(input)).toEqual(expectedOutput);
+	});
+
+	it('does set a h3 if there is more than one strong tag', () => {
```

Comment:
This might be left over from the move but this test has a duplicate name to the one above

```suggestion
	it('handles multiple strong tags where one contains an a tag inside it', () => {
```